internal/directory/google/google.go

```diff
@@ -49,14 +49,18 @@ func WithServiceAccount(serviceAccount *ServiceAccount) Option {
 
 // WithURL sets the provider url to use.
 func WithURL(url string) Option {
+	// if the empty string is sent in, use the default provider URL.
+	if url == "" {
+		url = defaultProviderURL
+	}
```

Comment:
imho, that makes `WithURL` less readable. 

I'd make this defaulting part of `New` instead, so that if you do not supply WithURL option, then a default would be used.

Reply:
I agree.